modules/installing-rhv-example-install-config-yaml.adoc

```diff
@@ -67,8 +67,10 @@ endif::openshift-origin[]
   - 172.30.0.0/16
 platform:
   ovirt:
-    api_vip: 10.46.8.230
-    ingress_vip: 192.168.1.5
+    api_vips: 
+      - 10.46.8.230
+    ingress_vips: 
+      - 192.168.1.5
```

Comment:
@creydr Where did we end up with VIPs outside the machine network for ovirt? It doesn't seem like this would work, but I guess they were running that way in ci.

Reply:
IIRC the ovirt people wanted to check their config first and remove the validation, if the validation is wrong: https://coreos.slack.com/archives/C68TNFWA2/p1663242301646269?thread_ts=1662377646.622669&cid=C68TNFWA2 - anyhow I haven't seen an exception for ovirt in the installer for the validation. 
@mburman5 how did you proceed on the `IP expected to be in one of the machine networks` issue on ovirt?

Reply:
I had a quick chat with @mburman5 and it seems they came over their CI issue, by adjusting their install-config

Reply:
@creydr So does that mean the VIPs do need to be on the same subnet as the machine network? If so, we'll need to update this example.

Reply:
It sounds like we should change this. Even if the installer weren't enforcing it (which it should, now), having the VIPs on separate subnets is not a best practice so we shouldn't document it.

Reply:
@cybertron - I have updated the IP addresses with your suggestion. Thanks!

Reply:
@cybertron - the last action I had was to update the IP address. Let me know if there is anything else or if you are ready to give your LGTM, thanks again!